sql/core/src/test/scala/org/apache/spark/sql/ConfigBehaviorSuite.scala

```diff
@@ -55,12 +54,12 @@ class ConfigBehaviorSuite extends QueryTest with SharedSparkSession {
 
     withSQLConf(SQLConf.SHUFFLE_PARTITIONS.key -> numPartitions.toString) {
       // The default chi-sq value should be low
-      assert(computeChiSquareTest() < 100)
+      assert(computeChiSquareTest() < 10)
```

Comment:
the physical plan is same as before, what caused this change?

Reply:
They are not same, we had two shuffles before, one was RoundRobinPartitioning, the other was RangePartitioning.

Reply:
ah i see